api/src/main/java/com/sciaps/ZAnalysisResult.java

```diff
@@ -0,0 +1,21 @@
+package com.sciaps;
+
+import java.util.List;
+
+public class ZAnalysisResult {
```

Comment:
what Z means?

Reply:
Z is the prefix for the LIBS family of devices and is consistent with our marketing literature and how customers know the products.  NGL is Z901, Z902 or Z903.  NGX is X505 and X550, so X will prefix classes for dealing with NGX devices.  The client will query the /id endpoint to get back a InstrumentId object from which it can tell it what kind of device it's talking to, based on the family field.